install/kubernetes/quick-install.yaml

```diff
@@ -561,10 +598,10 @@ spec:
         name: cilium-config-path
   updateStrategy:
     rollingUpdate:
-      maxUnavailable: 2
+      maxUnavailable: 
```

Comment:
is it OK to have no value for `maxUnavailable`?

Reply:
fixed, this spec has also been relocated to spec.strategy as can be seen in other templates, such as the etcd-operator deployment.